ci/Jenkinsfile.groovy

```diff
@@ -814,8 +815,6 @@ def test_in_docker(String testtag, String pyver, String docker_image, boolean la
     try {
         def docker_args = ""
         docker_args += "--rm --init "
-        docker_args += "-u `id -u`:`id -g` "
```

Comment:
hmm, why it was removed - i cannot guarantee that runnign docker under root will be supported in future.

Reply:
Getting error `PermissionError: [Errno 13] Permission denied: '/.local'` (http://mr-0xc1:8080/blue/organizations/jenkins/h2o-py-data-table/detail/PR-2287/23/pipeline/234)

I'm not saying that we should run under root, but whatever user is the default for docker, he must have sufficient permissions to do basic operations in the container.

Reply:
I've put the `-u` flag back, trying a different approach